acs-integration-tests/src/test/java/com/ge/predix/acs/cloudfoundry/PushMockedAdapterApplication.java

```diff
@@ -0,0 +1,94 @@
+package com.ge.predix.acs.cloudfoundry;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.Test;
+
+import com.ge.predix.cloudfoundry.client.CloudFoundryApplicationHelper;
+import com.ge.predix.cloudfoundry.client.CloudFoundryService;
+import com.ge.predix.cloudfoundry.client.CloudFoundryUtilities;
+
+@ContextConfiguration("classpath:integration-test-spring-context.xml")
+public final class PushMockedAdapterApplication extends AbstractTestNGSpringContextTests {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PushMockedAdapterApplication.class);
+
+    @Value("${ASSET_ADAPTER_ARTIFACT_DIR:../../acs-asset-adapter/target}")
+    private String assetAdapterArtifactDir;
+
+    @Value("${ASSET_ADAPTER_ARTIFACT_PATTERN:acs-asset-adapter-*.jar}")
+    private String assetAdapterArtifactPattern;
+
+    @Value("${ASSET_CLIENT_ID}")
+    private String assetClientId;
+
+    @Value("${ASSET_CLIENT_SECRET}")
+    private String assetClientSecret;
+
+    @Autowired
+    private CloudFoundryApplicationHelper cloudFoundryApplicationHelper;
+
+    @Autowired
+    private AcsCloudFoundryUtilities.Adapter adapterUtilities;
+
+    @Autowired
+    private DeleteApplications deleteApplications;
+
+    @Test(groups = { AcsCloudFoundryUtilities.Adapter.PUSH_ASSET_ADAPTER_APP_TEST_GROUP })
+    public void pushAssetAdapterApplication() throws Exception {
+        this.deleteApplications.deleteAssetAdapterApplication();
+
+        String assetUaaServiceName = "predix-uaa";
```

Comment:
Pull this out into a `static final` constant in `PushAdapterApplication.java` so that both `PushAdapterApplication` and this class can use it (since the service name should be the same between the adapter and its mock).